core/src/test/java/com/amplifyframework/logging/AndroidLoggingPluginTest.java

```diff
@@ -131,6 +137,39 @@ public void noContentLoggedAtThresholdNone() {
         assertTrue(systemLog.getLines().isEmpty());
     }
 
+    /**
+     * When the application is not in a debuggable build, logs are not stored.
+     */
+    @Test
+    public void noLogsStored() {
+        Context mockContext = mock(Context.class);
+        ApplicationInfo mockAppInfo = mock(ApplicationInfo.class);
```

Comment:
Do you need a mock for this one, or can you just create a real instance?

You probably do need a mock for the `context`.